### ISSUE TITLE
new inbound endpoint

### ISSUE THREAD

**Issue by max0783** [amz-tools/amazon-sp-api#276]
Hey there. I'm unable to version the new fulfillment inbound endpoint. Is this something that's still on the works? Thank you.

**Comment by amz-tools** [comment]
@max0783 Yes, we will include the endpoint soon. However if any new endpoints are not supported yet you always have the possibility to use `.callAPI()` with the `api_path` param which allows you to use endpoints that are not statically defined yet inside the library.

**Comment by stefan-soulutions** [comment]
hi, 
we are looking forward for the new release not only regarding the endpoint, but also the type definitions of response/replies. Can you estimate, when the new release will rolled out? Thx

**Comment by amz-tools** [comment]
@max0783 @stefan-soulutions Just released version 1.1.0 that includes the new `fulfillmentInbound` endpoint version.
